source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -299,6 +299,22 @@ AMControl::FailureExplanation AMPseudoMotorControl::calibrate(double oldValue, d
 	return AMControl::NoFailure;
 }
 
+void AMPseudoMotorControl::setMinimumValue(double newValue)
+{
+	if (minimumValue_ != newValue) {
+		minimumValue_ = newValue;
```

Comment:
Should there be a check here if the manually set override minimum is greater than any value which has come from epics? It seems like it might be a bad idea to allow acquaman to override a limit with a value which is less constraining than the one which comes from epics?

I mean, the motor still won't get there if we tell it to go beyond the limits. The setpoint will get pushed to epics, which will simply stop it at the limit value. Maybe an AMWarning?